lms/templates/footer.html

```diff
@@ -7,7 +7,7 @@
 <div class="wrapper wrapper-footer">
   <footer>
     <div class="colophon">
-      <nav class="nav-colophon">
+      <nav class="nav-colophon" aria-label="${_('Colophon')}">
```

Comment:
Rather than "Colophon", lets just call this "About" since it contains links about edX.